UnityProject/Assets/Scripts/Messages/Client/ClientMessage.cs

```diff
@@ -1,20 +1,15 @@
 ﻿using PlayGroup;
 using UnityEngine.Networking;
-using UnityEngine;
 
 public abstract class ClientMessage : GameMessageBase
 {
 	public NetworkInstanceId SentBy;
 
 	public void Send()
 	{
-		if (PlayerManager.LocalPlayer)
-		{
-			SentBy = LocalPlayerId();
-		}
-
+		SentBy = LocalPlayerId();
 		CustomNetworkManager.Instance.client.connection.Send(GetMessageType(), this);
-		Debug.LogFormat("Sent {0}", this);
+		//		Debug.LogFormat("Sent {0}", this);
```

Comment:
well I forgot to turn this debug off thanks for noticing, but still the rest shouldn't be reverted by your PR